backend/src/src-licenses/src/main/java/com/siemens/sw360/licenses/db/LicenseDatabaseHandler.java

```diff
@@ -370,7 +372,10 @@ public RequestStatus updateLicense(License inputLicense, User user, User request
 
             License dbLicense = new License();
             if(inputLicense.isSetId()) {
-                dbLicense = licenseRepository.get(inputLicense.getId());
+                License dbLicenseTmp = licenseRepository.get(inputLicense.getId());
```

Comment:
I'm not sure this temporary variable solves any real problem. Under what circumstances is it useful?